### ISSUE TITLE
No roster connected to Sakai site for combined kit

### ISSUE THREAD

**Issue by joliet** [DukeLearningInnovation/kits#429]
**Describe the bug**
The Sakai site that was created for the combined kit Foundations Elec & Comp Engineering on kits-test does not have the roster attached to the site.  Instead, the participants for the roster show as manually added to the site.  The problem with this is that these participants will not be managed by the SISS/Sakai process of updating the roster automatically through drop/add and other changes.

**To Reproduce**
1. Go to http://kits-test.oit.duke.edu
2. Navigate to COMBINED KIT Foundations Elec & Comp Engineering
2. Click the Sakai icon in the combined kit
4. Sakai opens to the correct site (ECE.110L.01.F19) but under Site Info, Roster(s) with Site Access is blank, and all participants show as manually added (they have the remove check box).

**Expected behavior**
When a Sakai site is setup for a combined kit, it should have at least one roster added to the site.  Ideally, it would have all the rosters of the combined kit added to the site.

**Screenshots**
![combined_kit_sakai_no_roster](https://user-images.githubusercontent.com/7387760/63098632-4dc44a80-bf41-11e9-86ae-027ed6fe978f.gif)

**Desktop (please complete the following information):**
 
- OS: macOS High Sierra Version: 10.13.6 (17G8030)
- Browser: Chrome
- Version: 76.0.3809.100 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

**Additional context**

**Comment by joliet** [comment]
Hi @ewendland23 .  Turns out I did create an issue for this one.  Instead of adding the participants across all rosters manually to the site, the preference would be to add the rosters with the providerID.  This happens when sites are created through batch, so maybe there's a way to get the rosters in the shared kits into the main kit?  Let me know how I can help here.

**Comment by joliet** [comment]
Is related to issue Created merged kits - https://github.com/DukeLearningInnovation/kits/issues/133

**Comment by joliet** [comment]
I just retested this on kits-test with instructor Clark Bray (cbray) and combined the following kits - MATH 216 01 Sp 20, MATH 216 02 Sp 20, MATH 216D 01 Sp20, MATH 216D 02 Sp20, MATH 216D 03 Sp20, MATH 216D 04 Sp20, MATH 216D 05 Sp20, MATH 216D 06 Sp20, MATH 216D 07 Sp20, and MATH 216D 08 Sp20.  I set up a Sakai site for the combined kit and the rosters were not added to the site, the students show up as individual members of the site and not connected to a roster.  The site is MATH.216.001.Sp20 and the Site ID is kits_shared.LinearAlgandDiffEqn.SHARED.1337.
